.travis.yml

```diff
@@ -13,9 +13,15 @@ env:
     - PUPPET_GEM_VERSION="~> 3.2.0"
     - PUPPET_GEM_VERSION="~> 3.3.0"
     - PUPPET_GEM_VERSION="~> 3.4.0"
-    - PUPPET_GEM_VERSION="~> 3.5.1"
+    - PUPPET_GEM_VERSION="~> 3.5.0"
     - PUPPET_GEM_VERSION="~> 3.6.0"
     - PUPPET_GEM_VERSION="~> 3.7.0"
+    - PUPPET_GEM_VERSION="~> 3.8.0"
+    - PUPPET_GEM_VERSION="~> 3" PARSER="future"
```

Comment:
I think we can just set 

`FUTURE_PARSER="yes"`

and then we don't need to modify the spec_helper.rb. This is what they are doing with puppetlabs/apache - https://github.com/puppetlabs/puppetlabs-apache/blob/master/.travis.yml